contracts/igp/interchain-gas-paymaster/src/interface.sw

```diff
@@ -0,0 +1,46 @@
+library interface;
+
+/// Logged when the benficiary is set.
+pub struct BeneficiarySetEvent {
```

Comment:
standard pattern to suffix with `Event`?

Reply:
I don't think there's an actual standard within Sway, just felt like a good way to differentiate structs that are just used for logging. wdyt?